src/components/discovery/LocusSearch.js

```diff
@@ -27,24 +26,29 @@ const LocusSearch = ({assemblyId, addVariantSearchValues, handleLocusChange, set
         const result = parse.exec(value);
 
         if (!result) {
-            setCurrentLocus({chrom: null, start: null, end: null});
             setLocusValidity(false);
```

Comment:
given the returns not actually mutating the locus now, should this validity set still be here? elsewhere it seems like `setLocusValidity` is being called outside `parsePosition`, and from an code comprehension POV it seems that making parsePosition not have any state side effects would be an improvement - then, this function could also be moved outside of `LocusSearch` to avoid redefining it every render

Reply:
yes, this side effect probably shouldn't be here. I think in fact it's redundant now that null values are passed to the parent component and then checked there.

Reply:
gone